scene/main/timer.cpp

```diff
@@ -65,7 +69,11 @@ void Timer::_notification(int p_what) {
 			if (!processing || timer_process_callback == TIMER_PROCESS_IDLE || !is_physics_processing_internal()) {
 				return;
 			}
-			time_left -= get_physics_process_delta_time();
+			if (ignore_time_scale) {
+				time_left -= Engine::get_singleton()->get_process_step();
```

Comment:
Shouldn't this use `get_physics_interpolation_fraction()` or whatever is physics frame equivalent?

Reply:
Hm, that would make sense, but that's not how `SceneTreeTimer`s do it - they appear to always use `Engine::get_singleton()->get_process_step()` when ignoring time scale, even for ones that process on the physics frame.

See https://github.com/godotengine/godot/blob/master/scene/main/scene_tree.cpp#L670

Could be a bug there, though?

Reply:
Nevermind, it's the same as SceneTreeTimer.